docs/misc/writing_docs.md

```diff
@@ -0,0 +1,351 @@
+# Tips For Writing Documentation
+
+Writing is easy. Writing _well_ is hard. Writing documentation is even harder and, as one might
+expect, writing _good documentation_ can be very hard. The challenge ultimately stems from the fact
+that good documentation requires attention to several things: consistency, clarity, detail,
+information density, unambiguity, and grammar (of course).
+
+This page describes tips for writing documentation (for M3 or otherwise). There are many different,
+equally-valid writing styles; this document isn't meant as a prescription for how to write, but
+instead as optional (but encouraged) guidelines geared toward writing consistent, clear,
+easy-to-read documentation.
+
+## Tip #1: Avoid first- and second-person pronouns.
+Documentation should always be written from a [third-person point of view][wp3p] (specifically,
+with a [third-person objective narrative voice][wp3ponv]); avoid using [first-][wp1p] and
+[second-person][wp2p]. The point of documentation is not to be a message from the author or to
+convey their point of view, but rather to be objective, factual, and canonical.
+
+| Examples | |
+| :-: | --- |
+| BAD | ~~You will need to set Value X to "foo", as our testing indicated that "bar" can cause issues when solar rays are present.~~ |
+| BAD | ~~We suggest setting Value X to "foo", as testing indicated that "bar" can cause issues when solar rays are present.~~ |
+| GOOD | **Value X must not be set to "bar", as testing indicated that it may cause issues when solar rays are present.** |
+| GOOD | **Value X must be set to "foo"; a value of "bar" may cause issues when solar rays are present.** |
+
+## Tip #2: Avoid subjective language.
+Subjective language – language that is open to interpretation, particularly based on perspective –
+should be avoided wherever possible. Documentation should state objective or broadly empirical
+truths (and should be supported by facts). Introducing perspective inherently creates ambiguity and
+vagueness in the document, as its meaning becomes reader-dependent. Importantly, cases of subjective
+language are distinct from cases of conditional behavior.
+
+| Examples | |
+| :-: | --- |
+| BAD | ~~Setting PowerLevel to 9000 is required, because efficiency is better than reliability in high-rate situations.~~ |
+| GOOD | **In high-rate situations (≥1k req/s), PowerLevel should be set to 9000 in order to maximize efficiency; otherwise, the system defaults to ensuring reliability.** |
+
+## Tip #3: Focus on concrete ideas.
+
```

Comment:
Either no line or line immediately after the `Tip`

Reply:
🙏